packages/interbit/src/tests/scripts/keys.test.js

```diff
@@ -26,7 +26,7 @@ describe('keys', () => {
 
     should.ok(file.publicKey)
     should.ok(file.privateKey)
-  })
+  }).timeout(5000)
 
```

Comment:
FYI, this is OK as long as there is just one test in the suite as the default suite timeout is also 5 secs.  But if you have two tests with 5 sec timeouts, or one test that requires a longer timeout, you will need to add `this.timeout(10000)` to `beforeEach`. (See `interbit.test.js`).